### ISSUE TITLE
CJ4 V 0.8.0 - chasing the track in NAV (Yaw damper?)

### ISSUE THREAD

**Issue by Jack737800** [Working-Title-MSFS-Mods/fspackages#638]
Today I did some flights with the new mod and first of all, it´s really good though :)

I noticed something, which is maybe related to the AP system, but I can not 100% tell you were this meight happen.
My obervings:

During climb and **cruise FL380 @ M.75** the track is almost the entire time not centered, thus results in a banking from the AP to try to get it centered again (I´ll upload some pictures so that you have an Idea what I´m talking about).
At the beginning (each wing hold about **1900lbs of fue**l), this banking is limited to about 2-3° left and right, but the AP is not able to center the needle.
With "only" **1400lbs** in each wing the track error and correctiv banking are getting larger! Now the AP is banking with about 7-9° left/right to get the needle centered. The needle shows about 1/2 dot deflection

![off the centerline](https://user-images.githubusercontent.com/71773282/102644496-36165b00-4161-11eb-823c-e8c758fd4d5f.jpg)

![centering the needle](https://user-images.githubusercontent.com/71773282/102644549-47f7fe00-4161-11eb-8fe8-f050d0b069b6.jpg)

![centering the needle 2](https://user-images.githubusercontent.com/71773282/102644650-707ff800-4161-11eb-8d7d-4af24d3a5b0d.jpg)

For my approach in KEWR I was decending to FL240 and at this point there was nothing at all...plane was flying the track without major changes (of course when there were some turbs.)

Also you can see in the pictures that the "YD" in the FMA is missing, but I did activate it after T/O! The panel tips show me only "press to deactivate yaw damper". I clicked here multiple times to make sure, I do not have forgotten it.

**Comment by nikilase** [comment]
+1 For the yaw dampener "sign" not shown when AP is on.

**Comment by cwburnett** [comment]
@nikilase @Jack737800 as with the real aircraft, YD is not annunciated when the AP is on. YD is forced on whenever AP is on, so it will always be on with the AP. You can confirm this by turning off the AP, at which point YD will be annunciated showing that it remains on. It should be turned off prior to landing.

As for the LNAV tracking we haven't observed this behavior, but will continue tweaking and improving over time.

**Comment by Jack737800** [comment]
I just did another long flight (about 5h) and this banking from left to right, when loosing weight is something I could see again.
I do have a little tailwind maybe 2-5 degress off with about 20 kts. Different as the first time I saw this, where the wind came from the side.
It happens always, when the plane is losing it´s desired track, even just only a bit (less then 1 dot), the AP starts to bank "aggressive" to get on it again.
Like I said, this phenomena is better so see, when the plane has burned some fuel.

![banking](https://user-images.githubusercontent.com/71773282/103689347-7cf2b600-4f93-11eb-9f15-2bfce4d7258f.jpg)
